webpages/settings/style.css

```diff
@@ -99,6 +99,10 @@ h1 {
   height: 100vh;
 }
 
+.addons-container.placeholder {
+  padding-top: 43px;
+}
+
```

Comment:
Instead of doing this, we could directly show the `<addon-group-header>` for the "new" group, which will probably end up being the first div in the container anyway (at least in stable releases)

Reply:
This is actually a UI/UX decision, not a technical one. UI/UX wise I prefer this. But just mentioning we could prerender the header for the new group instead of this just fine

Reply:
Yet another option is to show an empty group header.